src/generic/eigen_solver.h

```diff
@@ -238,6 +252,14 @@ namespace oomph
                             Vector<std::complex<double>>& eigenvalue,
                             Vector<DoubleVector>& eigenvector);
 
+    void solve_adjoint_eigenproblem(Problem* const& problem_pt,
+                                    const int& n_eval,
+                                    Vector<std::complex<double>>& eigenvalue,
+                                    Vector<DoubleVector>& eigenvector)
+    {
+      BrokenCopy::broken_assign("LAPACK_QZ");
+    };
```

Comment:
Resolved, as in above.

Reply:
Note that this could be causing some build fails on the development branch! (At least it does on my machine, after doing some fancy rebase things)

Reply:
The `oomph-lib` self-tests all pass fine so I'm not sure which build fails you're referring to. Can you provide some information?

Reply:
I don't know if it is a autotools thing or if I have done something I'm not supposed to, but every now and then `./autogen.sh` fails to build the library (after I've downloaded the latest commits say). After a `make maintainers-clean` followed by `./autogen.sh`, I am usually ok. However after the commit to remove the `Broken...`, I get build failed for several (not all the tests). For example, `demo_drivers/pml_helmholtz\scattering`. When I run `make unstructured_two_d_helmholtz`, I get the same issue as above:
``
/home/paul/Repositories/oomph-lib-jack/build/include/pml_helmholtz/complex_smoother.h:486:10: error: deleted function ‘void oomph::ComplexDampedJacobi<MATRIX>::solve(oomph::Problem* const&, oomph::DoubleVector&) [with MATRIX = oomph::CRDoubleMatrix]’ overriding non-deleted function
``
Should I put in an issue? Or can you take it from here?

Reply:
Oh boy, that's a silly mistake on my behalf. I've patched the bug and gone through the `delete`s again to make sure I haven't missed anymore and it looks like that was the only one (but I may be wrong!). Can you check out my changes and test the build again?
```bash
git checkout -b PuneetMatharu-patch-broken-build-due-to-cxx11-delete
git pull https://github.com/PuneetMatharu/oomph-lib.git patch-broken-build-due-to-cxx11-delete
```
Ordinarily, I would check this myself but the errors don't pop up on my end or on the automated self-tests, so I can't check them myself. Thanks!

Reply:
I've checked out the changes, and all the tests pass. Note that I don't have mpi at the moment but I don't think that should be a problem.
Do you have any idea why the tests would pass on your machine and not mine? This seems like it could be a fairly serious problem.

Reply:
Great, thanks for doing that. I had a further look at the self-tests (rather than just the green tick indicating it passed) and it appears some tests failed but the test still appears to "pass". I suspect it's an issue with the correct exit code not being propagated by the Python script we use to run the self-tests. I'll sort it out when I get the chance. A big thank you for spotting this nice and early!

Reply:
I guess we can resolve this conversation as it has been continued elsewhere.